### PR TITLE
Fix documentation typos and minor changes:

### PR DESCRIPTION
**Pull request by andradei**
- Fix some typos
- Some small changes to documentation to make it more clear
- Change the themes list from text to links
- Remove whitespace at the end of some lines

### CODE REVIEW

File: docs/content/documentation/content/section.md [hunk 1]
```diff
@@ -3,12 +3,10 @@ title = "Section"
 weight = 20
 +++
 
-A section is automatically created when a folder is found 
-in the `content` section, unless it only contains a `index.md` file and is actually
-a page with assets.
+A section is automatically created when a folder is found in the `content` section, unless it only
+contains a `index.md` file and is actuallya page with assets.
```
Comment (Keats) [inline docs/content/documentation/content/section.md]: space between actually and a

**Review by Keats — CHANGES_REQUESTED** [review]
Small typo but looks good otherwise, thanks!

File: docs/content/documentation/content/section.md (hunk 1 above)
Comment (andradei) [reply docs/content/documentation/content/section.md]: Good eye. Fixed.

**Comment by Keats** [comment]
Thanks!

**Pull request merged**